Octokit/Clients/DeploymentsClient.cs

```diff
@@ -34,10 +34,27 @@ public DeploymentsClient(IApiConnection apiConnection)
         /// <returns>All the <see cref="Deployment"/>s for the specified repository.</returns>
         public Task<IReadOnlyList<Deployment>> GetAll(string owner, string name)
         {
+            return GetAll(owner, name, ApiOptions.None);
```

Comment:
Like we mentioned earlier, can you bring back the `Ensure` arguments for `owner` and `name` here?